evaluation.py

```diff
@@ -237,7 +237,143 @@ def predict_keypoints(self,generator, location):
+            left_ankle_correct = []
+            right_ankle_correct = []
+
+            # Append True to correct joint list if distance is below threshold for any annotation
+            for j in range(annotations):
+                base = j*NUM_COCO_KEYPOINTS
+                nose_correct.append(dist_list[0+base]<=threshold)
+                left_eye_correct.append(dist_list[1+base]<=threshold)
+                right_eye_correct.append(dist_list[2+base]<=threshold)
+                left_ear_correct.append(dist_list[3+base]<=threshold)
+                right_ear_correct.append(dist_list[4+base]<=threshold)
+                left_shoulder_correct.append(dist_list[5+base]<=threshold)
+                right_shoulder_correct.append(dist_list[6+base]<=threshold)
+                left_elbow_correct.append(dist_list[7+base]<=threshold)
+                right_elbow_correct.append(dist_list[8+base]<=threshold)
+                left_wrist_correct.append(dist_list[9+base]<=threshold)
+                right_wrist_correct.append(dist_list[10+base]<=threshold)
+                left_hip_correct.append(dist_list[11+base]<=threshold)
+                right_hip_correct.append(dist_list[12+base]<=threshold)
+                left_knee_correct.append(dist_list[13+base]<=threshold)
+                right_knee_correct.append(dist_list[14+base]<=threshold)
+                left_ankle_correct.append(dist_list[15+base]<=threshold)
+                right_ankle_correct.append(dist_list[16+base]<=threshold)
+
+            # Add one to correct keypoint count if any annotation was below threshold for image
+            if True in nose_correct: correct_keypoints["nose"] += 1
+            if True in left_eye_correct: correct_keypoints["left_eye"] += 1
+            if True in right_eye_correct: correct_keypoints["right_eye"] += 1
+            if True in left_ear_correct: correct_keypoints["left_ear"] += 1
+            if True in right_ear_correct: correct_keypoints["right_ear"] += 1
+            if True in left_shoulder_correct: correct_keypoints["left_shoulder"] += 1
+            if True in right_shoulder_correct: correct_keypoints["right_shoulder"] += 1
+            if True in left_elbow_correct: correct_keypoints["left_elbow"] += 1
+            if True in right_elbow_correct: correct_keypoints["right_elbow"] += 1
+            if True in left_wrist_correct: correct_keypoints["left_wrist"] += 1
+            if True in right_wrist_correct: correct_keypoints["right_wrist"] += 1
+            if True in left_hip_correct: correct_keypoints["left_hip"] += 1
+            if True in right_hip_correct: correct_keypoints["right_hip"] += 1
+            if True in left_knee_correct: correct_keypoints["left_knee"] += 1
+            if True in right_knee_correct: correct_keypoints["right_knee"] += 1
+            if True in left_ankle_correct: correct_keypoints["left_ankle"] += 1
+            if True in right_ankle_correct: correct_keypoints["right_ankle"] += 1
```

Comment:
And then here we can just say `if right_knee_correct: correct_keypoints[right_knee] += 1`